qdk/environment.yml

```
@@ -1,19 +1,18 @@
 name: qdk
 channels:
-  - rdkit
-  - quantum-engineering
   - conda-forge
+  - quantum-engineering
```

`quantum-engineering` only hosts the `qsharp` and `iqsharp` packages, but has since been deprecated in favor of the `microsoft` channel.
```suggestion
  - microsoft
```